crates/sui/src/client_commands.rs

```diff
@@ -1495,6 +1499,23 @@ impl SuiClientCommands {
                     recovery_phrase: phrase,
                 })
             }
+
+            SuiClientCommands::RemoveAddress { alias_or_address } => {
+                let address: SuiAddress = match context
+                    .config
+                    .keystore
+                    .get_address_by_alias(alias_or_address.clone())
+                {
+                    Ok(addr) => *addr,
+                    Err(_) => SuiAddress::from_str(&alias_or_address)
+                        .map_err(|e| anyhow!("Invalid value: {}", e))?,
```

Comment:
```suggestion
                        .map_err(|e| anyhow!("Invalid address or alias: {}", e))?,
```

Just to give a little more context on the error.